helm/templates/clusterrole-proteus-aws-operator-metrics-reader.yaml

```diff
@@ -1,9 +1,15 @@
-apiVersion: rbac.authorization.k8s.io/v1
-kind: ClusterRole
-metadata:
-    name: proteus-aws-operator-metrics-reader
-rules:
-    - nonResourceURLs:
-        - /metrics
-      verbs:
-        - get
+# {{ if eq .Values.installScope "cluster" }}
```

Comment:
Is a reader role still required?

Reply:
Good catch, this is not required anymore and don't see any requirements of bindings in here, hence deleted.